python/openql.i

```diff
@@ -545,9 +547,15 @@ arg1 : str
 arg2 : []
     list of qubits
 arg3 : int
-    duration in ns (at the moment it is only supported for wait instruction, in the future it will be extended to override duration of any gate)
-arg3 : double
-    angle of rotation, used internally only for rotations (rx, ry and rz)
+    duration in ns (default: 0)
+arg4 : double
+    angle of rotation, used internally only for rotations (rx, ry and rz) (default: 0.0)
+arg5 : []
```

Comment:
I'll add condgate in C++ interface as in Python interface; I forgot to add it.
But will keep the current gate() interface as well because an all-inclusive interface is the ultimate fallback
and having all kinds of variations I want to avoid.

Reply:
Added condgate.